client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/style.scss

```diff
@@ -296,6 +296,13 @@ $font-family: "SF Pro Text", $sans;
 		color: var(--color-neutral-60);
 	}
 
+	/**
+	 * The header of the stepper framework
+	 */
+	.signup-header {
+		z-index: 10; // Increase the z-index to avoid the WordPress Logo being covered by the sidebar
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)
Are there any standards (such as CSS variables) about `z-index` across WordPress.com?
I'm asking because I think `z-index` often has conflicts with another `z-index`.

Reply:
Calypso has this file https://github.com/Automattic/wp-calypso/blob/trunk/client/assets/stylesheets/shared/functions/_z-index.scss where we can better plan out our z-indexes 🙂

Reply:
Thank you for letting me know!
I see that there are other things in `shared`. I understand it for now!